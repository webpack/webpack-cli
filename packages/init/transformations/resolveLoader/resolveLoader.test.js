"use strict";

const defineTest = require("@webpack-cli/utils/defineTest");

defineTest(
	__dirname,
	"resolveLoader",
	"resolveLoader-0",
	{
		modules: ["'ok'", "mode_nodules"],
		mainFields: ["no", "'main'"],
		moduleExtensions: ["'-kn'", "ok"]
	},
	"init"
);

defineTest(
	__dirname,
	"resolveLoader",
	"resolveLoader-1",
	{
		modules: ["'ok'", "mode_nodules"],
		mainFields: ["no", "'main'"],
		moduleExtensions: ["'-kn'", "ok"]
	},
	"add"
);
