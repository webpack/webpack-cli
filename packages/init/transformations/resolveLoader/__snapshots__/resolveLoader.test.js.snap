// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`resolveLoader transforms correctly using "resolveLoader-0" data 1`] = `
"module.exports = {
    entry: 'index.js',

    output: {
		filename: 'bundle.js'
	},

    resolveLoader: {
        modules: ['ok', mode_nodules],
        mainFields: [no, 'main'],
        moduleExtensions: ['-kn', ok]
    }
}
"
`;

exports[`resolveLoader transforms correctly using "resolveLoader-1" data 1`] = `
"module.exports = {
	entry: 'index.js',
	output: {
		filename: 'bundle.js'
	},
	resolveLoader: {
		moduleExtensions: ['-loader', '-kn', ok]
	}
}
"
`;
